main.go

```diff
@@ -159,6 +160,12 @@ func main() {
 		}
 	}
 
+	// Retrieve the option to enable HTTP2 on the Webhook server
+	enableWebhookHTTP2 := os.Getenv("ENABLE_WEBHOOK_HTTP2")
```

Comment:
Nit: we can probably move this down to the if webhooks enabled block down below, so that we don’t need to bother retrieving this when webhooks are disabled

Reply:
updated!